### ISSUE TITLE
Validation accuracy keeps to be 0.09% during training

### ISSUE THREAD

**Issue by edizhuang** [cheerss/CrossFormer#5]
Dear authors,

I'm interested in your paper and perfom training from scratch on ImageNet. However, the validation accuracy keeps to be * Acc@1 0.090 during training. 

Do you have any idea why this happens? I train Swin Transformer, it works.

I use Pytorch 1.7.1 and 1.6.0, no mixed precision, 100 epochs.

--amp-opt-level O0  --output ./output --opts TRAIN.EPOCHS 100

Thanks,
Eddie

**Comment by edizhuang** [comment]
Hi cheerss,

It does work with --amp-opt-level native. The first epoch is * Acc@1 1.488.

It is just so wired that only mixed precision work. Just let you know.

Thanks,
Eddie
